Add deprecation message to trafficserver.set_var function

rallytime opened the pull request:
The `set_var` function has a deprecation notice in the documentation only for the `trafficserver.set_var` function, but no warning at the CLI/logs. This PR adds a "warn_util" message.

Since there haven't been any warnings to the user about the removal of this function, the version is being bumped from Oxygen to Fluorine so users have time to adapt as needed.

Commit:
Add deprecation message to trafficserver.set_var function

The set_var function has a deprecation notice in the documentation
only for the trafficserver.set_var function, but no warning at the
CLI/logs. This PR adds a "warn_util" message.

Since there haven't been any warnings to the user about the removal
of this function, the version is being bumped from Oxygen to Fluorine
so users have time to adapt as needed.

salt-jenkins commented:
@rallytime, thanks for your PR! By analyzing the history of the files in this pull request, we identified @jpeach, @cedwards and @multani to be potential reviewers.

rallytime commented:
I just realized that the trafficsever execution module has a couple of functions without the `warn_until` notation as well. I'll add another commit to handle those as well.

Commit:
Add deprecation messages to the trafficserver execution module

This commit also includes a PY3 compatibility/pylint fix for the state module

The import for salt.utils was also fixed and all references in this state
were updated.

rallytime commented:
Go Go Jenkins!
